### ISSUE TITLE
Publish version of boost that will install dependencies that have up to date peer deps?

### ISSUE THREAD

**Issue by jgbpercy** [apollographql/apollo-angular#933]
Hello!

Loving Apollo and Apollo Angular, thanks for the great work.

I have an existing Angular project that uses apollo-angular-boost. I've just updated to Angular 7 and graphql 14, which results in these peer dependency warnings:
```
npm WARN apollo-angular-link-http@1.2.0 requires a peer of @angular/core@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-angular-link-http@1.2.0 requires a peer of @angular/common@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-angular-link-http@1.2.0 requires a peer of graphql@^0.11.3 || ^0.12.3 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-angular-link-http-common@1.2.0 requires a peer of @angular/core@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-angular-link-http-common@1.2.0 requires a peer of @angular/common@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-angular-link-http-common@1.2.0 requires a peer of graphql@^0.11.3 || ^0.12.3 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
```

Seems like apollo-angular-link-http and -common have newer versions with compatible peer deps, but boost doesn't have a version that forces these to be installed. Is that in the works? Or is there some way to make npm install more up to date sub dependencies that I'm not aware of?

Thanks!

**Comment by kamilkisiela** [comment]
Thanks :) I released v1.4.0

**Comment by jgbpercy** [comment]
Awesome, thanks so much for the quick response :)
